templates/vue/src/components/Lightbox.vue

```diff
@@ -201,6 +201,12 @@ export default {
 }
 </script>
 
+<style>
+.lightbox-open {
```

Comment:
To avoid conflicts, please add `tapestry` at the beginning of your class name. i.e. `tapestry-lightbox-open` instead of `lightbox-open`. Even better, put `body.tapestry-lightbox-open` instead so it only applies to the body.